api/models/record.js

```diff
@@ -58,6 +58,36 @@ const Record = BaseModel.extend({
       return this.related('trial').url;
     }
   },
+  trialsPerRegistry: function () {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('primary_register as registry'),
+      bookshelf.knex.raw('count(primary_register)::int')
+    )
+    .from('trialrecords')
+    .groupBy('primary_register')
+    .orderBy('primary_register')
+    .then((rows) => {
+      return rows;
+    });
+  },
+  lastRegistryUpdate: function () {
+  return bookshelf.knex
+    .select(
+      bookshelf.knex.raw('source_id as id'),
+      bookshelf.knex.raw('sources.name as name'),
+      bookshelf.knex.raw('max(trialrecords.updated_at) as updatedate')
+    )
+    .from('trialrecords')
+    .leftJoin('sources', 'source_id', 'sources.id')
+    .groupBy('source_id', 'sources.name')
```

Comment:
when i put ID only i got error :
`Unhandled rejection error: column "sources.name" must appear in the GROUP BY clause or be used in an aggregate function`